src/order/lattice.lean

```diff
@@ -356,6 +356,24 @@ by apply_instance
 
 end lattice
 
+namespace monotone
+
+open lattice
+
+variables {α : Type u} {β : Type v}
+
+lemma map_sup [semilattice_sup α] [semilattice_sup β]
```

Comment:
Maybe rename this `le_map_sup` (and `map_inf` to `map_inf_le`). I would expect that `map_sup` says something like "a lattice morphism preserves `sup`".

Reply:
Oh, I'm just seeing that this is in the namespace `monotone`. Now I'm ambivalent about making this change, so feel free to ignore it.

Reply:
Renamed.